backend/kernelCI_app/views/treeDetailsSlowView.py

```diff
@@ -41,15 +40,15 @@ def __init__(self):
         self.testFailReasons = {}
         self.testArchSummary = {}
         self.testIssues = []
-        self.testEnvironmentCompatible = defaultdict(int)
+        self.testEnvironmentCompatible = {}
         self.bootHistory = []
         self.bootStatusSummary = {}
         self.bootConfigs = {}
         self.bootPlatformsFailing = set()
         self.bootFailReasons = {}
         self.bootArchSummary = {}
         self.bootIssues = []
-        self.bootEnvironmentCompatible = defaultdict(int)
+        self.bootEnvironmentCompatible = {}
```

Comment:
you can use nested default dict by passing it as a lambda 
I think it is something like this `defaultdict(lambda: defaultdict(int))`